octoprint_findmymrbeam/static/js/findmymrbeam_settings_viewmodel.js

```diff
@@ -61,7 +73,20 @@ $(function () {
             self.verifyByFrontend();
         };
 
-        self.verifyByFrontend = function() {
+        self.sendToExtension = function () {
+            if (self.enabled()) {
+                // send to find.mr-beam extension's contentScript
+                // TODO: should be done whenever searchID changes...
```

Comment:
You could have `searchID` [subscribe] to `sendToExtension`

[subscribe]: https://knockoutjs.com/documentation/observables.html#explicitly-subscribing-to-observables